ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -301,30 +312,32 @@ export default class ConfirmTransactionBase extends Component {
       />
     );
 
+    const showInlineControls = process.env.SHOW_EIP_1559_UI
```

Comment:
There was a slight issue wit hthe previous logic for showing these fields. after London we'll *only* show the inline fields if they have that setting enabled.